airflow/settings.py

```diff
@@ -101,7 +101,7 @@
 """
 
 engine: Engine
-Session: Callable[..., SASession]
+Session: sessionmaker
```

Comment:
This change is not needed, 
```suggestion
Session: Callable[..., SASession]
```